mods/ra/maps/fall-of-greece-1-personal-war/rules.yaml

```diff
@@ -70,7 +70,7 @@ TRUK:
 
 HEALCRATE:
 	Crate:
-		Lifetime: 0
+		Duration: 0
```

Comment:
These mission overrides were consolidated into campaign-rules.yaml which will need to be changed to `Duration`.